File: src/queries/GetCharacters.ts
import { gql } from "@apollo/client";

export default gql`
  query GetCharacters($page: Int!, $filter: FilterCharacter!) {
    characters(page: $page, filter: $filter) {
      info {
        next
      }
      results {
        id
        name
        status
        gender
        image
      }
    }
  }
`;

File: rickandmortyflutter/lib/query/get_characters.dart
String getCharactersQuery = """
  query GetCharacters(\$page: Int!) {
    characters(page: \$page) {
      info {
        next
      }
      results {
        id
        name
        status
        gender
        image
        episode {
          id
          name
          episode
        }
      }
    }
}
""";
